python/plugins/processing/algs/qgis/ui/FieldsMappingPanel.py

```diff
@@ -199,7 +213,12 @@ def data(self, index, role=Qt.DisplayRole):
             return hAlign + Qt.AlignVCenter
 
         if role == Qt.BackgroundRole:
-            return QBrush(QColor(255, 224, 178)) if 'not_null' in field and field['not_null'] else QVariant()
+            return QBrush(QColor(255, 224, 178)) if 'constraints' in field and field['constraints'] else QVariant()
+
+        if role == Qt.ToolTipRole:
+            if column_def['name'] == 'constraints':
+                return ", ".join(
```

Comment:
```suggestion
                return "<br> ".join(
```

Would this make it more readable?

Reply:
I think so. Applied on a single commit with the other suggestion. (This one shows an Outdated tag and it is not possible to commit it from GitHub)